src/coreComponents/constitutive/contact/ContactBase.hpp

```diff
@@ -60,8 +70,7 @@ class ContactBaseUpdates
   GEOSX_HOST_DEVICE
   inline
   virtual real64 computeEffectiveAperture( real64 const aperture,
-                                           real64 & dEffectiveAperture_dAperture ) const
-  { GEOSX_UNUSED_VAR( aperture, dEffectiveAperture_dAperture ); return 0.0; }
+                                           real64 & dEffectiveAperture_dAperture ) const;
```

Comment:
should we be using the term "aperture" any longer?

Reply:
I think `hydraulicAperture` may be a good term. I also don't love the fact that this `apertureTable` is now owned by the contact model but, at the moment, I can't think of a better location for it. This has more to do with how flow properties of the fractures are computed than with the contact model.